force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -349,20 +451,30 @@ public with sharing class FlowControlLoopsCollections {
         // Initialize a Map
         Map<String, Integer> employeeSalaries = new Map<String, Integer>();
 
+
+        employeeSalaries.put('John Doe', 50000);
+        employeeSalaries.put('Jane Smith', 60000);
+        employeeSalaries.put('Sam Brown', 55000);
+        employeeSalaries.put('Alice Johnson', 65000);
         // Add key-value pairs to the Map
```

Comment:
It's more advanced but try using the same variable employeeSalraies for question 19 and 20.

Reply:
This will require you making a static variable outside both methods.